dll/ntdll/def/ntdll.spec

```diff
@@ -1365,7 +1365,7 @@
 1351 cdecl memcpy(ptr ptr long) memmove
 1352 cdecl memmove(ptr ptr long)
 1353 cdecl memset(ptr long long)
-1354 cdecl -arch=i386,x86_64 pow(double double)
+1354 cdecl -arch=i386,x86_64,arm pow(double double)
```

Comment:
Is there any architecture that doesn't support this function? Otherwise just remove all arch things?

Reply:
There is an x86 implementation,
and stubs for x64 and arm